yarn.lock

```diff
@@ -960,10 +960,10 @@ loose-envify@^1.0.0, loose-envify@^1.4.0:
   dependencies:
     js-tokens "^3.0.0 || ^4.0.0"
 
-mantine-datatable@^2.8.5:
```

Comment:
Actually, why do we need this second lock file?

Reply:
just `./src/frontend/yarn.lock`, not sure where you are getting package-lock.json files with yarn